cli/lib/metadata.ts

```diff
@@ -54,6 +54,8 @@ export async function injectMetadata(this: Context, jobUri: string) {
 
   const propertyShapes = new TermMap<QuadSubject, QuadObject>()
 
+  const creatorTerms = dataset.pointer.out(dcterms.creator).terms
```

Comment:
Do if the "opendata.swiss organization" is not set in the cube metadata, observations won't have an `observedBy`. Is that ok? I thought `observedBy` was mandatory?

From the discussion last week, I thought the idea was to store the value for `observedBy` on the publishing profile?

Reply:
Ah, maybe. @l00mi ?